src/components/LogBookHistoryRow.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+import LogBookHistoryRow from '../interfaces/LogBookHistoryRow';
+
+const LogBookHistoryRowComp: FC<LogBookHistoryRow> = (props) => {
+  return (
+    <tr>
+      <td>{props.historyRow.divingCylinder}</td>
+      <td>
+        {props.historyRow.compressedAir != null
+          ? props.historyRow.compressedAir
+          : 0}{' '}
+        bar
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator
 
```suggestion
{props.historyRow.compressedAir ??  0}{' '}bar
```